api/handler/v1beta1/appeal.go

```diff
@@ -166,20 +176,25 @@ func (s *GRPCServer) CancelAppeal(ctx context.Context, req *guardianv1beta1.Canc
 	}, nil
 }
 
-func (s *GRPCServer) listAppeals(ctx context.Context, filters *domain.ListAppealsFilter) ([]*guardianv1beta1.Appeal, error) {
+func (s *GRPCServer) listAppeals(ctx context.Context, filters *domain.ListAppealsFilter) ([]*guardianv1beta1.Appeal, int64, error) {
 	appeals, err := s.appealService.Find(ctx, filters)
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to get appeal list: %s", err)
+		return nil, 0, status.Errorf(codes.Internal, "failed to get appeal list: %s", err)
+	}
+
+	total, err := s.appealService.GetAppealsTotalCount(ctx, filters)
```

Comment:
can we make these two calls run in parallel using `errgroup`?

Reply:
okay will try